### ISSUE TITLE
Issue with get_cis_benchmark statement

### ISSUE THREAD

**Issue by thecosmictrickster** [Teutenberg/DBAid#27]
Line 230:
SELECT '2.17','2.17 Ensure no login exists with the name ''sa'' (Scored)' AS [Policy Name], CASE WHEN EXISTS(SELECT [name] FROM [master].[sys].[server_principals] WHERE [name] = 'sa') THEN 0 ELSE 1 END AS [score], CASE WHEN EXISTS(SELECT [name] FROM [master].[sys].[server_principals] WHERE [name] = 'sa') THEN CAST('sa' AS NVARCHAR(128)) ELSE '0' END AS [value]

Found edge case where for some reason it fails with "error converting varchar 'sa' to int". Only seen it once in many implementations. The fix seems to be to change the return value data type of the first CASE statement to varchar (i.e. wrap the values in single quotes):
SELECT '2.17','2.17 Ensure no login exists with the name ''sa'' (Scored)' AS [Policy Name], CASE WHEN EXISTS(SELECT [name] FROM [master].[sys].[server_principals] WHERE [name] = 'sa') THEN '0' ELSE '1' END AS [score], CASE WHEN EXISTS(SELECT [name] FROM [master].[sys].[server_principals] WHERE [name] = 'sa') THEN CAST('sa' AS NVARCHAR(128)) ELSE '0' END AS [value]

**Comment by thecosmictrickster** [comment]
Fix applied, waiting on merge into master.

**Comment by thecosmictrickster** [comment]
Fix merged with master
